include/bluetooth/iso.h

```diff
@@ -165,6 +151,45 @@ struct bt_iso_recv_info {
 	uint8_t flags;
 };
 
+
+/** Opaque type representing an Broadcast Isochronous Group (BIG). */
+struct bt_iso_cig;
+
+struct bt_iso_cig_create_param {
+	/** Array of pointers to CIS channels */
+	struct bt_iso_chan **cis_channels;
+
+	/** Number channels in @p cis_channels */
+	uint8_t num_cis;
+
+	/** Channel interval in us. Value range 0x0000FF - 0x0FFFFFF. */
+	uint32_t interval;
+
+	/** Channel Latency in ms. Value range 0x0005 - 0x0FA0. */
+	uint16_t latency;
+
+	/** @brief Channel peripherals sleep clock accuracy Only for CIS
```

Comment:
Please specify the unit.

Reply:
The unit is described the values in the enum (`BT_GAP_SCA_*`)

Reply:
Made a reference to the BT_GAP_SCA* macros.